cypress/page_objects/allGalleries.js

```diff
@@ -0,0 +1,100 @@
+class AllGalleries {
+    get inputField() {
+        return cy.get(".form-control");
+    }
+
+    get filterBtn() {
+        return cy.get(".input-button");
+    }
+
+    get loadMoreBtn() {
+        return cy.get(".btn-custom");
+    }
+
+    get firstGallery() {
+        return cy.get(".box-title").eq(0);
+    }
+
+    get secondGallery() {
+        return cy.get(".box-title").eq(2);
+    }
+
+    get thirdGallery() {
+        return cy.get(".box-title").eq(4);
+    }
+
+    get gallery4() {
+        return cy.get(".box-title").eq(6);
+    }
+
+    get gallery5() {
+        return cy.get(".box-title").eq(8);
+    }
+
+    get gallery6() {
+        return cy.get(".box-title").eq(10);
+    }
+
+    get gallery7() {
+        return cy.get(".box-title").eq(12);
+    }
+
+    get gallery8() {
+        return cy.get(".box-title").eq(14);
+    }
+
+    get gallery9() {
+        return cy.get(".box-title").eq(16);
+    }
+
+    get gallery10() {
+        return cy.get(".box-title").eq(18);
+    }
+
+    get authorOfFirstGallery() {
+        return cy.get(".box-title").eq(1);
+    }
+
```

Comment:
Ovde bismo upotrebili `this` kljucnu rec, odnosno ne bi pisali poseban getter za detalje na kartici galerije, nego bismo iskoristili vec napisan getter za galeriju i unutar njega trazili element koji nam treba, na primer: `this.firstGallery.find('.box-title')`;